contracts/src/v0.8/keystone/CapabilityRegistry.sol

```diff
@@ -19,6 +19,19 @@ contract CapabilityRegistry is OwnerIsCreator, TypeAndVersionInterface {
     string name;
   }
 
+  struct Node {
```

Comment:
How about signer address to verify signatures on observations/don2don messages?

Reply:
Signer address makes sense. Anything else?